server/devpi_server/model.py

```diff
@@ -149,6 +149,15 @@ def getstage(self, user, index=None):
             return user.getstage(index)
 
 
+def ensure_boolean(value):
+    if isinstance(value, bool):
```

Comment:
check if we can make treatment of yes or true symetric to no or false

Reply:
Changed in https://github.com/devpi/devpi/pull/664/commits/a94d47ad323d7a8a81945da093b0a53ecf88dabf